documents/Strategies/Building strategy.tex

```diff
@@ -0,0 +1,39 @@
+\documentclass{article}
+
+\usepackage{graphicx}
+
+\author{Michiel van den Berg}
+\title{Strategy for building buildings}
+
+\begin{document}
+\pagenumbering{gobble}
+
+\maketitle{}
+\newpage{}
+
+\section{Introduction}
+
+In this document the strategy about the decision of building buildings and which buildings will be explained. In the first section the reason to build a building will be defined, and in the second section the choice for a particular building will be clarified. This document will be used to implement the following user story:\\
+\\
+\textit{"As an agent\\
+When in a simulation\\
+I am able to build stuff if I think 
+this is benificial for me\\
+To optimize a certain indicator."\\}
+
+\section{When/why to build buildings}
+In this section the reasons to build a building will be explained. The first reason to build a building, is that it can influence an indicator in a positive way. Another reason why to build something, is that a building can give you profit in terms of your budget.
+
+
+\subsection{Direct influence}
+There is one direct reason to build a building. We created an indicator that has the goal to make high buildings. The more high buildings, the better the indicator will be. But there is a downside with building something. Building a building costs a lot of money and the budget is not unlimited. The agent will build stuff as long as the budget holds and as long as it gives a positive influence on the indicators.
+
```

Comment:
Building new buildings usually has a positive effect on the budget, so this isn't a real problem. Only destroying buildings costs a lot of money.

Also, there are two building indicators. One of them just needs more Education space, and the other one also needs high buildings. Maybe write something about this in this paragraph.